server/apierror/error.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2018 Palantir Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apierror
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/palantir/go-baseapp/baseapp"
+)
+
+type ErrorResponse struct {
+	Error string `json:"error"`
+}
+
+func WriteAPIError(w http.ResponseWriter, code int, message ...string) error {
```

Comment:
Yeah it can, it was in a separate package because it was being used by the `middleware` originally too, but now that we are performing the token check in the handler there is no need for it to be in a separate package. Will consolidate.